src/MSBuild/app.config

```diff
@@ -33,7 +33,7 @@
         <!-- Redirects for assemblies redistributed by MSBuild (in the .vsix). -->
         <dependentAssembly>
           <assemblyIdentity name="Microsoft.Bcl.AsyncInterfaces" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-          <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0" />
+          <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
```

Comment:
I see it was previously bumped with the bunch of other packages
https://github.com/dotnet/msbuild/commit/352c48c0217a664f9190272ff69163f7295bded4#diff-af84f50fd5aa2e8bfc3479cb6ddf7882bc084ce1af01de6dd49ff93aa2d1718cR111

@rainersigwald , could you please confirm if it's safe to do it only for Microsoft.Bcl.AsyncInterfaces?

Reply:
What package needs this? (similarly for M.E.Primitives and M.E.Configuration.Binder)

Reply:
OpenTelemetry 1.10.0 I think, I will test in the WIP branch if I can downgrade to 1.9.0. But I don't see why not take 9.0 https://github.com/dotnet/msbuild/pull/9535 upgraded them in ~December

Reply:
downgraded